src/Fluid/Flu_FixUp_Flux.cpp

```diff
@@ -292,6 +292,19 @@ void Flu_FixUp_Flux( const int lv )
                                                      CorrVal[ENGY], CorrVal+NCOMP_FLUID,
                                                      CheckMinTemp_No, NULL_REAL, Emag, EoS_DensEint2Temp_CPUPtr,
                                                      EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table );
+
+//                check output temperature initial guess
+#                 ifdef GAMER_DEBUG
+                  if (  Hydro_CheckUnphysical( UNPHY_MODE_SING, &CorrVal[TEMP_IG], "output temperature initial guess", ERROR_INFO, UNPHY_VERBOSE )  )
+                  {
+                     printf( "   CorrVal[DENS]=%13.7e code units, CorrVal[MOMX]=%13.7e code units, CorrVal[MOMY]=%13.7e code units, CorrVal[MOMZ]=%13.7e code units, CorrVal[ENGY]=%13.7e code units\n",
+                                 CorrVal[DENS], CorrVal[MOMX], CorrVal[MOMY], CorrVal[MOMZ], CorrVal[ENGY] );
+                     printf( "Passive:" );
+                     for (int v=0; v<NCOMP_PASSIVE; v++)  printf( " [%d]=%13.7e", v, CorrVal[NCOMP_FLUID+v] );
+                     printf("\n");
+                  }
+#                 endif // #ifdef GAMER_DEBUG
+
 #                 endif
```

Comment:
Add `// if ( EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )`

Reply:
I have added the comment.